ingestion/cadet.yaml

```diff
@@ -6,9 +6,7 @@ source:
     manifest_path: "s3://mojap-derived-tables/prod/run_artefacts/latest/target/manifest.json"
     catalog_path: "s3://mojap-derived-tables/prod/run_artefacts/latest/target/catalog.json"
     test_results_path: "s3://mojap-derived-tables/prod/run_artefacts/latest/target/run_results.json"
-    platform_instance: cadet
     target_platform: athena
-    target_platform_instance: athena_cadet
```

Comment:
I think we should retain the platform instances. Should we have different dbt sources in future it'll be useful to distinguish them.